lib/day10.ex

```diff
@@ -18,9 +85,31 @@ defmodule Day10.Part1 do
       tiles
       |> Enum.find(fn {char, _row, _col} -> char == "S" end)
 
-    process_tiles(tiles, [start], [start])
+    pipes = process_tiles(tiles, [start], [start])
 
-    div(length(process_tiles(tiles, [start], [start])), 2)
+    Enum.filter(tiles, fn tile -> tile not in pipes end)
+    |> Enum.map(fn not_pipe ->
+      num_intersects =
+        Enum.filter(tiles, fn {_, row, col} ->
+          elem(not_pipe, 1) == row and elem(not_pipe, 2) > col
+        end)
+        |> Enum.reduce(0, fn t, acc ->
+          case {Enum.member?(pipes, t), elem(t, 0)} do
+            {true, "S"} -> acc + 1
```

Comment:
Adding this case causes a few tests to fail, but it gives me the correct answer with the puzzle output. The correct solution would be to convert this to the corresponding pipe character based on the nearby pipes, but I don't want to do that at this moment so I'm going to leave it for now.